docs/config.md

```diff
@@ -505,6 +505,20 @@ log = "main@origin.."
 The default value for `revsets.log` is
 `'present(@) | ancestors(immutable_heads().., 2) | present(trunk())'`.
 
+### Prioritize Revsets in the Log over @
+
+In some situations the default graph can be hard to read, for example when working with big merges.
+To improve this behaviour you can configure which revset in the `jj log` graph is displayed on
```

Comment:
Looking at the repo, the US spelling "behavior" seems more prevalent than the UK one "behaviour" (even if it's still present). Perhaps the US one should be preferred?

```suggestion
To improve this behavior you can configure which revset in the `jj log` graph is displayed on
```